specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,321 @@
+= Spec - Introducing a new component monitoring-integration
+
+Remove existing performance monitoring component and replace it with a new
+monitoring-integration component.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
+responsibilities would be:
+
+* monitoring-integration should store predefined json’s for dashboards, panels and alerts.
+* Keeping track of the topology changes in etcd to configure the grafana dashboards
```

Comment:
i have mentioned that in under second milestone section
https://github.com/Tendrl/specifications/pull/218/files#diff-339c615a84970e1b496c5433f62b799f

Flow structure for that may be covered in different spec in second milestone